robottelo/cli/imprt.py

```diff
@@ -0,0 +1,341 @@
+# -*- encoding: utf-8 -*-
```

Comment:
not a typo but a reserved word.
I was just unable to come up with anything better so far